tests/functional/test_application_restricted.py

```diff
@@ -422,6 +422,7 @@ def patch_patient(auth: dict, context: dict):
         {
             "Content-Type": "application/json-patch+json",
             "If-Match": patient_version_id,
+            "X-Request-ID": str(uuid.uuid4()),
```

Comment:
The PATCH request was using the same Request ID header as the previous GET request. Request IDs are expected to be unique to each request and so this is assumed to be the reason behind test failures.